package.json

```diff
@@ -27,18 +27,20 @@
     "write-changelog": "./node_modules/.bin/conventional-changelog -i CHANGELOG.md -s -p angular"
   },
   "dependencies": {
-    "angular2": "2.0.0-beta.3",
-    "es6-promise": "3.0.2",
+    "angular2": "2.0.0-beta.7",
+    "es6-promise": "^3.0.2",
     "es6-shim": "0.33.3",
     "es7-reflect-metadata": "1.5.5",
-    "rxjs": "5.0.0-beta.0",
-    "zone.js": "0.5.10"
+    "marked": "^0.3.5",
+    "ng2-contentful": "git+https://git@github.com/dydek/ng2-contentful.git#79f983a",
```

Comment:
I know that but I'd been changing it many times and before final version ( it's still not the final version, even if we are talking about beta ) , it was easier to use it without publishing to npm repository. I'll move it when it'll be ready for first beta release